.github/workflows/code_formatting_checks.yml

```diff
@@ -0,0 +1,55 @@
+name: Code formatting
+
+# Scheduled at 11:30 PM UTC (05:00 AM IST)
+on:
+  schedule:
+    - cron: "30 23 * * *"
```

Comment:
Since the minute is 0-based, this will actually run at 05:01 AM IST and not 05:00 AM.

Reply:
Never mind, this is my mistake.